scripts/run_e2e_tests.py

```diff
@@ -184,12 +184,6 @@ def cleanup():
         common.kill_processes_based_on_regex(p)
     build.set_constants_to_default()
 
-    if not common.is_windows_os():
```

Comment:
Wouldn't you still need memcache to be alive for e2e tests? They all run on a single server which is initialized just once at the outset before running any of the tests.

Reply:
Done. I realized this after seeing the e2e tests fail again.